packages/replay-next/components/sources/log-point-panel/LogPointPanel.module.css

```diff
@@ -4,14 +4,23 @@
   flex-direction: column;
   padding: 0.5rem;
   gap: 0.25rem;
-
   --badge-picker-button-size: 1.25rem;
   --badge-picker-icon-size: 1rem;
 }
 .PanelEnabled {
   background-color: var(--point-panel-background-color);
 }
 
+.PanelDisabled {
+  background: repeating-linear-gradient(
+    45deg,
+    var(--theme-base-100),
+    var(--theme-base-100) 5px,
+    var(--point-panel-background-color) 5px,
+    var(--point-panel-background-color) 10px
```

Comment:
This looks like a bad mix of CSS variables. Since `--point-panel-background-color` is detached from the `--theme-base-*` colors, it could change in a way that makes this not work. We should probably add a new `--point-panel-background-color*` for this.

Reply:
Is it important that we use 5px increments here rather than our normal 0.25rem increments? I'd prefer the latter if possible.

Reply:
Moot point for now, since we're moving away from stripes!